src/pages/drop/token/[id].tsx

```diff
@@ -0,0 +1,72 @@
+import { Badge, Box } from '@chakra-ui/react';
+
+import { DropManager } from '@/modules/DropManager/DropManager';
+
+interface TokenDropResponse {
+  name: string;
+  links: {
+    id: number;
+    link: string;
+    hasClaimed: boolean;
+  }[];
+}
+
+const tableColumns = [
+  { title: 'Link', selector: (row) => row.link },
```

Comment:
the row.link 's color is different from figma

Reply:
<img width="241" alt="image" src="https://user-images.githubusercontent.com/19740800/210783724-50907477-43c5-47ef-9d81-5ac78968f07f.png">
good catch!